src/Sql/dbo/Tables/ProviderOrganizationProviderUser.sql

```diff
@@ -0,0 +1,11 @@
+﻿CREATE TABLE [dbo].[ProviderOrganizationProviderUser] (
+    [Id]                     UNIQUEIDENTIFIER    NOT NULL,
+    [ProviderOrganizationId] UNIQUEIDENTIFIER    NOT NULL,
+    [ProviderUserId]         UNIQUEIDENTIFIER    NULL,
+    [Type]                   TINYINT             NOT NULL,
+    [CreationDate]           DATETIME2 (7)       NOT NULL,
```

Comment:
Will likely need permissions here that are specific to a Provider User and what they can do with this specific Org within this relationship.

Reply:
Do we still want to keep `Type` here or only custom permissions?

Reply:
Yes, and add Permissions